.github/workflows/ci.yaml

```diff
@@ -15,6 +15,10 @@ env:
   PRE_COMMIT_CACHE: ~/.cache/pre-commit
   SQLALCHEMY_WARN_20: 1
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
Will this not also cancel the concurrent builds in dev?

Reply:
Yes. Which is fine.